RHIDP-6504: Modularize modules/dynamic-plugins/proc-topology-configure (#1065)
redhat-developer/red-hat-developers-documentation-rhdh: modules/dynamic-plugins/proc-viewing-openshift-routes.adoc

[id="proc-viewing-openshift-routes_{context}"]
= Viewing OpenShift routes

> To view OpenShift routes, you must grant read access to the routes resource in the Cluster Role:
> 
> [source,yaml]
> ----
>   apiVersion: rbac.authorization.k8s.io/v1
>   kind: ClusterRole
>   metadata:
>     name: backstage-read-only
>   rules:
>     ...
>     - apiGroups:
>         - route.openshift.io
>       resources:
>         - routes
>       verbs:
>         - get
>         - list
> ----
> 
> You must also add the following in `kubernetes.customResources` property in your `{my-app-config-file}` file:
> 
> [source,yaml]

Reviewer: ```suggestion
.Procedure
. To view OpenShift routes, grant read access to the routes resource in the Cluster Role:
+
[source,yaml]
----
  apiVersion: rbac.authorization.k8s.io/v1
  kind: ClusterRole
  metadata:
    name: backstage-read-only
  rules:
    ...
    - apiGroups:
        - route.openshift.io
      resources:
        - routes
      verbs:
        - get
        - list
----

. Also add the following in `kubernetes.customResources` property in your `{my-app-config-file}` file:
+
[source,yaml]
```
